libcontainer/intelrdt/intelrdt.go

```diff
@@ -298,20 +312,26 @@ func isIntelRdtMounted() bool {
 	return true
 }
 
-func parseCpuInfoFile(path string) (bool, bool, error) {
-	isCatFlagSet := false
-	isMbaFlagSet := false
+type cpuInfoFlags struct {
+	isCatFlagSet      bool
+	isMbaFlagSet      bool
+	isMbmTotalFlagSet bool
+	isMbmLocalFlagSet bool
+}
+
+func parseCpuInfoFile(path string) (cpuInfoFlags, error) {
+	infoFlags := cpuInfoFlags{false, false, false, false}
```

Comment:
You don't have to explicitly set `bool` fields to `false` -- it is a default value.